recipes/jaeger-browser/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "jaeger-browser" %}
+{% set name_ = "jaeger_browser" %}
```

Comment:
Just put these in the URL, no need for a variable that won't be used elsewhere.

Reply:
sounds great. I am changing that now. thanks!